projects/storefrontapp-e2e-cypress/cypress/helpers/asm.ts

```diff
@@ -200,6 +229,15 @@ export function asmTests() {
         checkout.signOutUser();
       });
     });
+
+    describe('End Session Button', () => {
+      it('should display the asm menu for an agent when ?asm=true is passed into the url', () => {
+        checkout.visitHomePage('asm=true');
+        cy.get('cx-asm').should('exist');
+        cy.get('cx-asm-main-ui').should('exist');
+        cy.get('cx-csagent-login-form').should('exist');
+      });
+    });
```

Comment:
Looks like a duplicate of the test we have on line 28 
`agent should see the asm UI when ?asm=true is passed to the url'`